Druid/DRPrimitiveIRGenerator.class.st

```diff
@@ -1,3 +1,8 @@
+"
+Expects a primitive method, this is, a method of an interpreter implementing a primitive (e.g. #primitiveAdd) and answer an IR, useful to generate code.
```

Comment:
```suggestion
Expects a primitive method, this is a method of an interpreter implementing a primitive (e.g. #primitiveAdd), and generates a Druid's IR.
```

Reply:
I changed "answer" to "generates" because this produces an effect (and I like to use "answer" queries)